### PR TITLE
German corpus and ref data

### PR DESCRIPTION
**Pull request by Garrett-Reed**
This will create the German corpus, ref tables, and ref data

### CODE REVIEW

File: SQL-Scripts/create_German_ref_data.sql [hunk 1]
```diff
@@ -0,0 +1,64 @@
+/* Change Log															*/
+/* Beta - 4/4/2021 - Garrett Reed - Initial Creation - Create German Reference Tables and Data	*/
```
Comment (oldosamaibraheem11) [inline SQL-Scripts/create_German_ref_data.sql:2]: Test comment

File: SQL-Scripts/pop_German_corpus.sql [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+/* Instructions	*/
+/* You will need the corpus on your local machine as well.	*/
+/* Set the filepath to whatever it should be for your local machine.	*/
+/* You may need to add these two lines to your Advanced DB connection settings	*/
+/* AllowLoadLocalInfile=true	*/
+/* OPT_LOCAL_INFILE=1	*/
+
+
+SET GLOBAL local_infile = true; 
+LOAD DATA LOCAL INFILE "C:\\Users\\Garth\\Documents\\GitHub\\CSCI5030\\Corpora\\deu-de_web-wrt_2019_100K-sentences_tagged.txt" INTO table german_corpus
```
Comment (myadlapalli) [inline SQL-Scripts/pop_German_corpus.sql]: We need to change the table in the load statement to TABLE.

Comment (Garrett-Reed) [reply SQL-Scripts/pop_German_corpus.sql]: Will update and re-request tomorrow.

File: SQL-Scripts/create_German_ref_data.sql [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+use wordsense;
+
+/* Create table structures */
+DROP TABLE IF EXISTS German_Index;
+DROP TABLE IF EXISTS German_Corpus;
```
Comment (oldosamaibraheem11) [inline SQL-Scripts/create_German_ref_data.sql:8]: Good! In case the table is ￼corrupt

**Review by oldosamaibraheem11 — CHANGES_REQUESTED** [review]
Looks good to me! We just need to remove the big file from the pull request. Since as a team we decided to not have those in the repo.￼

File: SQL-Scripts/create_German_ref_data.sql (hunk 3 above)
Comment (myadlapalli) [reply SQL-Scripts/create_German_ref_data.sql:8]: Looks Good! We can add to check status of table to handle if table corrupted

**Review by oldosamaibraheem11 — APPROVED** [review]

**Pull request merged**